lib/pure/math.nim

```diff
@@ -202,28 +202,39 @@ when not defined(JS):
     ## computes x to power raised of y.
     
   # C procs:
-  proc srand(seed: cint) {.importc: "srand", header: "<stdlib.h>".}
-  proc rand(): cint {.importc: "rand", header: "<stdlib.h>".}
+  when defined(windows):
+    # The "secure" random, available from Windows XP
+    # https://msdn.microsoft.com/en-us/library/sxtz2fa8.aspx
+    proc rand_s(val: var cuint) {.importc: "rand_s", header: "<stdlib.h>".}
```

Comment:
Are you sure even MingW has this in its headers? Smells Visual Studio specific to me.